src/NodeDynamics/CurrentSourceInverterMinimal.jl

```diff
@@ -0,0 +1,30 @@
+@doc doc"""
+```Julia
+CSIMinimal(;I_r)
+```
+
+A node type that keeps the current fixed as a desired values `I_r`.
+
+`CSIMinimal` models an inverters as an ideal current source. This can be the
+most simple representation of an inverter in grid-feeding mode, according to
+Rocabert, Joan, et al. "Control of power converters in AC microgrids." (2012).
+Here, additionally to `u`, there are no internal dynamic variables.
+
+
+# Keyword Arguments
+- `I_r`: reference/ desired current
+
+
+# Mathematical Representation
+Using `CSIMinimal` for node ``a`` gives:
+```math
+0 = I_{r,a} - \left\|i_a\right\|
+```
+
+"""
+@DynamicNode CSIMinimal(I_r) <: OrdinaryNodeDynamics() begin
```

Comment:
Shouldn't that be an ODE with mass?

Reply:
thanks! good point. I changed that.

Reply:
erm, where? there is no commit?

Reply:
@SabineAuer are you going to add the change commit?

Reply:
@timkittel: I commited my changes and had some issues with master being merged into this branch, I think.
Now, there is [some coverage issue](https://travis-ci.org/JuliaEnergy/PowerDynBase.jl/jobs/509689162#L644)  again during the travis CI check?

Reply:
No, your `CSIMinimal` test fails (:

(If the tests fail, the coverage reports are not even generated.)

Reply:
Thanks! The error message did not help me out. Should have checked the tests. Now they pass :)